packages/qwik/src/optimizer/src/plugins/eslint-plugin.ts

```diff
@@ -51,6 +51,9 @@ export async function createLinter(
 
         report.forEach((file) => {
           for (const message of file.messages) {
+            if (message.ruleId != null && !message.ruleId.startsWith('qwik/')) {
```

Comment:
It seems `ruleId` can be `null` when it's some eslint configuration error. I assume we still want to show those.